README.md

```diff
@@ -2,92 +2,133 @@
+The Portal Network is divided into the following sub-protocols.
 
-The Portal Network aims to solve this problem by designing our networks so that each additional client that joints the network adds additional capacity to the network.  The end result *should* be a network which becomes more robust and powerful as more nodes join the network.
+- State Network
+- History Network
+- Transaction Gossip Network
+- Header Gossip Network
+- Canonical Indices Network
 
-> Additional reading: https://snakecharmers.ethereum.org/the-winding-road-to-functional-light-clients/
-> 
-> Additional watching: https://www.youtube.com/watch?v=MZxqRs_tLNs
+Each of these networks is designed to deliver a specific unit of functionality.  Most portal clients will participate in all of these networks in order to deliver the full JSON-RPC API.  Each network however is designed to be independent of the others, allowing clients the option of only participating in a subset of them if they wish.
 
+All of the sub-protocols in the Portal Network establish their own Kademlia DHT network that is managed independent of the base Discovery V5 protocol Kademlia DHT.
 
-### JSON-RPC API
+
+## Design Principles
+
+Each of the Portal Network sub-protocols follows these design principles.
+
+1. Isolation
+  - Each network should be able to be run independent of the other networks.  Participation in one network should not require participation in another network.
+2. Distribution of Responsibility
+  - Normal operation of the network should result in a roughly even spread of responsibility across the individual nodes in the network.
+3. Tunable Resource Requirements
+  - Individual nodes should be able to control the amount of machine resources (HDD/CPU/RAM) they must expend to be part of the network.
+
+These design principles are aimed at ensuring that participation in the Portal Network is feasible even on resource constrained devices.
+
+## The JSON-RPC API
 
 The following JSON-RPC API endpoints are intended to be supported by the portal network and exposed by portal clients.
 
 TODO
 
+## Bridge Nodes
+
+We use the term "bridge" node to refer to the process by which data enters the Portal Network.  A "bridge" node is a normal node in the Portal Network that is able to bring in new and missing data from some external source and inject it into the Portal Network.  These nodes would not be special in any way from the perspective of the Portal Network.
+
+The planned architecture for bridge nodes is to pull data from the standard JSON-RPC API of a Full Node and "push" this data into their respective networks within the Portal Network.
+
 ## Network Functionality
 
-### State: Accounts and Contract Storage
+### State Network: Accounts and Contract Storage
+
+The State Network facilitates on-demand retrieval of the Ethereum "state" data.  This includes things like:
```

Comment:
s/this includes things like/this includes/

I think what follows is the full list!